File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/admin/AdminOperationsAction.kt
package com.hhp227.concafe.presentation.main.admin

sealed interface AdminOperationsAction {
    data object ClickNotifications : AdminOperationsAction
    data object ClickSeeAllPending : AdminOperationsAction
    data object ClickBannerRegister : AdminOperationsAction
    data class SelectPendingFilter(val filter: PendingFilter) : AdminOperationsAction
    data class ApprovePending(val id: String) : AdminOperationsAction
    data class RejectPending(val id: String) : AdminOperationsAction
    data class ClickQuickMenu(val id: String) : AdminOperationsAction
    data object DismissInfoMessage : AdminOperationsAction
}

File: iosApp/iosApp/Presentation/Main/Admin/AdminOperationsAction.swift
//
//  AdminOperationsAction.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/07.
//

import Foundation

enum AdminOperationsAction {
    case clickNotifications
    case clickSeeAllPending
    case clickBannerRegister
    case loadMoreInquiries
    case loadMoreReports
    case selectPendingFilter(PendingFilter)
    case approvePending(String)
    case rejectPending(String)
    case clickQuickMenu(String)
    case dismissInfoMessage
}
